com.playeveryware.eos/Runtime/Core/Config/EOSConfig.cs

```diff
@@ -200,12 +327,14 @@ protected EOSConfig() : base("EpicOnlineServicesConfig.json") { }
         /// If set to false, the plugin will attempt to delay the input to
         /// mitigate CPU spikes caused by spamming the SDK.
         /// </summary>
+        [ConfigField("Send input without delay", 
+            ConfigFieldType.Flag, 
+            "Workaround to force send input without any delay. If " +
+            "true, the native plugin will always send input receive directly " +
+            "to the SDK.", 4)]
         public bool hackForceSendInputDirectlyToSDK;
 
-        /// <summary>
-        /// Set to 'true' if the application is a dedicated game server.
-        /// </summary>
-        public bool isServer;
```

Comment:
while this _looks_ to be removed, it was in-fact **moved** to an earlier place within the class definition.